packages/libs/react-ui/src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,43 @@
+import { accordionWrapperClass } from './Accordion.css';
+import { AccordionSection, IAccordionSection } from './AccordionSection';
+
+import React, { FC, useState } from 'react';
+
+export interface IAccordionProps {
+  className?: string;
```

Comment:
Right now I don't think it is getting applied to any of the JSX though?

In general we are trying to keep it more strict. This will result in more difficulty when trying to do something specific with components, but the hope is that it will result in more conformity. In what instances have you needed to change button styles or text fields?